subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -9694,17 +9760,22 @@ static bool is_enc_req_pause_tx(struct connection *conn)
 	pdu_data_tx = (void *)conn->pkt_tx_head->pdu_data;
 	if ((pdu_data_tx->ll_id == PDU_DATA_LLID_CTRL) &&
 	    (pdu_data_tx->llctrl.opcode == PDU_DATA_LLCTRL_TYPE_ENC_REQ)) {
-		if ((conn->llcp_req != conn->llcp_ack) ||
+		if (((conn->llcp_req != conn->llcp_ack) &&
+		     (conn->llcp_type != LLCP_ENCRYPTION)) ||
+		    ((conn->llcp_req == conn->llcp_ack) &&
```

Comment:
The following statements mean that "no procedure is active, but procedures are pending" In which we would actually start the procedure?